docs/group-user-management.rst

```diff
@@ -228,6 +243,34 @@ The administrator or administrator with restricted privileges can add an LDAP us
 
 #. Note that it is advisable to clarify the OMERO group membership situation of the LDAP users soon after their joined OMERO. This can be done for example by adding the new user to their lab group (e.g. ``Lab1``) in OMERO as well and changing the default group of such user in OMERO to be their lab group. See above for how to change the default group of a user. Otherwise, the new LDAP&OMERO users might be importing their data into the ``My Data`` group for some period of time, without realizing the data are not accessible (because ``My Data`` is a private group) to their colleagues in the lab group for cooperative purposes.
 
+*Set up OMERO server to email users*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are a full administrator of OMERO, you can email OMERO users.
```

Comment:
Can't light admins do it?

Reply:
Very hard to test. From the top of my head, following deliberation

As this is kind of hard to define feature, it has been erred on the side of safety and thus the full admin flag was given.

But then again, thinking about the fact that maybe it just takes an access to the Admin tab...

Reply:
Tested with @mtbc - the truth is, even light admin with least privileges can do it.

Reply:
Fixed in https://github.com/ome/omero-guide-introduction/pull/6/commits/23890f7366d48c7721f0f4535a26754a14313c4b